src/Bookings.js

```diff
@@ -1,21 +1,65 @@
-import React from "react";
+
+import React, {useState,useEffect} from "react";
+
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+// import FakeBookings from “./data/fakeBookings.json”;
+
 
 const Bookings = () => {
+  const [bookings, setBookings] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me`).then(res =>
+      res.json().then(data => {
+        console.log(data);
+        setBookings(data);
+      })
+    );
+  }, []);
```

Comment:
Really nice - as a small improvement you can actually extract that `const [bookings, setBookings] = useState([]);useEffect(() => { fetch(...) }, [])` logic into a custom hook for extra style points ;)